pkg/orchestration/wiring/k8scompute/plugin.go

```diff
@@ -420,6 +445,65 @@ func buildContainers(spec *Spec, envDefault []core_v1.EnvVar, envFrom []core_v1.
 	return containers
 }
 
+func buildAffinity(resourceName voyager.ResourceName) *core_v1.Affinity {
+	return &core_v1.Affinity{
+		PodAntiAffinity: buildAntiAffinity(resourceName),
+	}
+}
+
+func buildAntiAffinity(resourceName voyager.ResourceName) *core_v1.PodAntiAffinity {
+	matchExpressions := []meta_v1.LabelSelectorRequirement{
+		meta_v1.LabelSelectorRequirement{
+			Key:      "app",
+			Operator: "In",
+			Values: []string{
+				string(resourceName),
+			},
+		},
+	}
+
+	// Create WeightedPodAffinityTerms to configure antiaffinity to distibute
+	// the app to different zones, and then nodes (where possible)
+	podAffinityTerms := []core_v1.WeightedPodAffinityTerm{
+		core_v1.WeightedPodAffinityTerm{
+			Weight: 100,
```

Comment:
Using a weight of 100 makes it impossible for us to add another singular rule which trumps this one.

What's the advantage?

Reply:
Per discussion, have reduced this to `75`.